frontend/controller/app/group.js

```diff
@@ -62,7 +62,13 @@ export default (sbp('sbp/selectors/register', {
     sbp('state/vuex/commit', 'setCurrentGroupId', { contractID: groupId, isNewlyCreated })
   },
   'gi.app/group/joinAndSwitch': async function (params: $Exact<ChelKeyRequestParams>) {
-    await sbp('gi.actions/group/join', params)
+    await sbp('gi.actions/group/join', params).catch(e => {
+      sbp('gi.ui/prompt', {
+        heading: L('Failed to join the group'),
+        question: L('Error details:{br_}{err}', { err: e.message, ...LTags() }),
+        primaryButton: L('Close')
+      })
+    })
```

Comment:
Before we didn't have an error handler here, which means the caller would be responsible for logging the error.

But now with the `.catch` we have two problems:

1. The error is no longer logged, and we need it logged in the logs, not just displayed via the prompt.
2. Now if an error occurs, we will switch to a group we couldn't join (line 73)

Reply:
> 1. The error is no longer logged, and we need it logged in the logs, not just displayed via the prompt.

Not entirely accurate, before this code was in `actions`.

> Now if an error occurs, we will switch to a group we couldn't join (line 73)

True, this should be fixed.